src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst

```diff
@@ -39,4 +39,4 @@ Job Control
   * `scontrol update timelimit=4:00:00 jobid=12345` change the walltime of the job
   * `scontrol update jobid=12345 dependency=afterany:54321` only start the job after job with id `54321` has finished
   * `scontrol hold jobid=12345` prevent the job from starting
-  * `scontrol unhold jobid=12345` release the job to be eligible for run (after it was set on hold)
+  * `scontrol uhold jobid=12345` release the job to be eligible for run (after it was set on hold)
```

Comment:
`scontrol --help` on tells me it's `scontrol release <job_list>`.

I am also not sure you really need to write `jobid=` in the examples above...

```
$ scontrol --help
scontrol [<OPTION>] [<COMMAND>]                                            
    Valid <OPTION> values are:                                             
     -a or --all: equivalent to "all" command                            
     -d or --details: equivalent to "details" command                    
     -h or --help: equivalent to "help" command                          
     --hide: equivalent to "hide" command                                
     -M or --cluster: equivalent to "cluster" command                    
     -o or --oneliner: equivalent to "oneliner" command                  
     -Q or --quiet: equivalent to "quiet" command                        
     -v or --verbose: equivalent to "verbose" command                    
     -V or --version: equivalent to "version" command                    
									   
  <keyword> may be omitted from the execute line and scontrol will execute 
  in interactive mode. It will process commands as entered until explicitly
  terminated.                                                              
									   
    Valid <COMMAND> values are:                                            
     abort                    shutdown slurm controller immediately        
			      generating a core file.                      
     all                      display information about all partitions,    
			      including hidden partitions.                 
     cluster                  cluster to issue commands to.  Default is    
			      current cluster.  cluster with no name will  
			      reset to default.                            
     checkpoint <CH_OP><ID>   perform a checkpoint operation on identified 
			      job or job step 
     completing               display jobs in completing state along with  
			      their completing or down nodes               
     create <SPECIFICATIONS>  create a new partition or reservation        
     details                  evokes additional details from the "show"  
			      command                                      
     delete <SPECIFICATIONS>  delete the specified partition or reservation
			      On Dynamic layout Bluegene systems you can also
			      delete blocks.                               
     errnumstr <ERRNO>        Given a Slurm error number, return a         
                              descriptive string.                          
     exit                     terminate scontrol                           
     help                     print this description of use.               
     hold <job_list>          prevent specified job from starting. <job_list>
			      is either a space separate list of job IDs or
			      job names 
     holdu <job_list>         place user hold on specified job (see hold)  
     hide                     do not display information about hidden      
			      partitions                                   
     listpids <job_id<.step>> List pids associated with the given jobid, or
			      all jobs if no id is given (This will only   
			      display the processes on the node which the  
			      scontrol is ran on, and only for those       
			      processes spawned by SLURM and their         
			      descendants)                                 
     notify <job_id> msg      send message to specified job                
     oneliner                 report output one record per line.           
     pidinfo <pid>            return slurm job information for given pid.  
     ping                     print status of slurmctld daemons.           
     quiet                    print no messages other than error messages. 
     quit                     terminate this command.                      
     reboot_nodes [<nodelist>]  reboot the nodes when they become idle.    
                              By default all nodes are rebooted.           
     reconfigure              re-read configuration files.                 
     release <job_list>       permit specified job to start (see hold)     
     requeue <job_id>         re-queue a batch job                         
     requeuehold <job_id>     re-queue and hold a batch                    
     resume <jobid_list>      resume previously suspended job (see suspend)
     setdebug <level>         set slurmctld debug level                    
     setdebugflags [+|-]<flag>  add or remove slurmctld DebugFlags         
     schedloglevel <slevel>   set scheduler log level                      
     show <ENTITY> [<ID>]     display state of identified entity, default  
			      is all records.                              
     shutdown <OPTS>          shutdown slurm daemons                       
			      (the primary controller will be stopped)     
     suspend <job_list>       susend specified job (see resume)            
     top <job_id>             Put specified job first in queue for user    
     takeover                 ask slurm backup controller to take over     
     uhold <jobid_list>       place user hold on specified job (see hold)  
     update <SPECIFICATIONS>  update job, node, partition, reservation,    
			      step or bluegene block/submp configuration   
     verbose                  enable detailed logging.                     
     version                  display tool version number.                 
     wait_job <job_id>        wait until the nodes allocated to the job    
			      are booted and usable                        
     !!                       Repeat the last command entered.             
									   
  <ENTITY> may be "aliases", "assoc_mgr" "burstBuffer",              
       "config", "daemons", "frontend",                              
       "hostlist", "hostlistsorted", "hostnames",                    
       "job", "layouts", "node", "partition", "reservation",     
       "slurmd", "step", or "topology"                               
       (also for BlueGene only: "block" or "submp").                   
									   
  <ID> may be a configuration parameter name, job id, node name, partition 
       name, reservation name, job step id, or hostlist or pathname to a   
       list of host names.                                                 
									   
  <HOSTLIST> may either be a comma separated list of host names or the     
       absolute pathname of a file (with leading '/' containing host names 
       either separated by commas or new-lines                             
									   
  <LEVEL> may be an integer value like SlurmctldDebug in the slurm.conf    
       file or the name of the most detailed errors to report (e.g. "info",
       "verbose", "debug", "debug2", etc.).                          
									   
  <SLEVEL> may be an integer value like SlurmSchedLogLevel in the          
       slurm.conf file or "enable" or "disable".                       
									   
  <OPTS> may be "slurmctld" to shutdown just the slurmctld daemon,       
       otherwise all slurm daemons are shutdown                            
									   
  Node names may be specified using simple range expressions,              
  (e.g. "lx[10-20]" corresponds to lx10, lx11, lx12, ...)                
  The job step id is the job id followed by a period and the step id.      
									   
  <SPECIFICATIONS> are specified in the same format as the configuration   
  file. You may wish to use the "show" keyword then use its output as    
  input for the update keyword, editing as needed.  Bluegene blocks/submps 
  are only able to be set to an error or free state.  You can also remove  
  blocks by specifying 'remove' as the state.  The remove option is only   
  valid on Dynamic layout systems.                                         
  (Bluegene systems only)                                                  
									   
  <CH_OP> identify checkpoint operations and may be "able", "disable", 
  "enable", "create", "vacate", "requeue", "restart", or "error"
  Additional options include "ImageDir=<dir>", "MaxWait=<seconds>" and 
  "StickToNodes"   
									   
  All commands and options are case-insensitive, although node names and   
  partition names tests are case-sensitive (node names "LX" and "lx"   
  are distinct).
```